### DISCUSSION TITLE
How to change the color of the dark mode toggle in light mode?

### DISCUSSION THREAD

**Discussion by maelle** [quarto-dev/quarto-cli#8380] [Q&A]
### Description

Yet again a dark mode question :sweat_smile:

The dark mode toggle is a "background-image". I wonder whether there is an easy way to change its color, with a css rule or option? Thank you!

**Comment by mcanouil** [comment]
The following might be of interest[^1], especially `fill="#{colorToRGBA($navbar-light-color)}"`:

- https://github.com/quarto-dev/quarto-cli/blob/037736475e1b0b61573306581eb109b44d3ab578/src/resources/formats/html/bootstrap/_bootstrap-rules.scss#L1643-L1658

[^1]: I did not try to play with this for now. The colour might actually be updated by JavaScript on click on the toggle button.

CC @dragonstyle might already know the trick here.

**Reply by dragonstyle** [reply]
That SCSS variable should be respected - the theme CSS should control this (e.g. the dark and light theme will each provide that scss variable). If you find otherwise let us know as this should be customizable.

We could consider adding a specific color for that toggle, if useful.

**Comment by maelle** [comment]
Thanks both, and sorry for the delay!
I was expecting the color of the toggle, and of the search icon actually, to both follow `$nav-link-color`. Does that make any sense?

**Reply by mcanouil** [reply]
I'm not sure to follow which part should follow which variable exactly.
